examples/Weather Station/with_sleep_mode.ino

```diff
@@ -0,0 +1,50 @@
+/* This example contains the arduino code for 
+weather station with sleep mode clearly 
+depicting the use case of LowPower library.
+Go through the without_sleep_mode.ino code 
+for a better understanding of it's working.*/
+
+#include <dht.h>
+#include <LowPower.h>
```

Comment:
This library doesn't contain a file named LowPower.h

Reply:
Oh damn, so sorry for the confusion sir. It was a mistake on my part. Really sorry, will be more careful from next time.